### ISSUE TITLE
Trade Republic Date Bug

### ISSUE THREAD

**Issue by codeofsumit** [codeofsumit/tresor-import#17]
A user reported that his Trade Rep PDF is not importing correctly.
Tried it before and it seems to be an issue with the date.
This is the `textArr` from his PDF.

```javascript
[
  "TRADE REPUBLIC BANK GMBH",
  "KASTANIENALLEE 32",
  "10435 BERLIN",
  "Muster Bla",
  "Lalala",
  "D 22222 Irgendwo",
  "SEITE",
  "1",
  "von",
  "1",
  "DATUM",
  "16.01.2020",
  "AUSFÜHRUNG",
  "72b6-3605",
  "SPARPLAN",
  "bf90-b7f9",
  "DEPOT",
  "192132301",
  "Trade Republic Bank GmbH",
  "Kastanienallee 32",
  "10435 Berlin",
  "www.traderepublic.com",
  "service@traderepublic.com",
  "Sitz der Gesellschaft: Düsseldorf",
  "AG Düsseldorf HRB 85864",
  "USt-ID DE307510626",
  "Geschäftsführer",
  "Ingo Hillen",
  "Karsten Müller",
  "ABRE / 2020-01-16 / 71478627 / 11cc-82b7",
  "WERTPAPIERABRECHNUNG SPARPLAN",
  "ÜBERSICHT",
  "Sparplanausführung am 16.01.2020 an der Lang & Schwarz Exchange.",
  "Der Kontrahent der Transaktion ist Lang & Schwarz TradeCenter AG & Co. KG.",
  "POSITION",
  "ANZAHL",
  "KURS",
  "BETRAG",
  "iShsII-Core MSCI Europe U.ETF",
  "Registered Shares o.N.",
  "ISIN: IE00B1YZSC51",
  "1,3404 Stk.",
  "26,11 EUR",
  "35,00 EUR",
  "GESAMT",
  "35,00 EUR",
  "BUCHUNG",
  "VERRECHNUNGSKONTO",
  "VALUTA",
  "BETRAG",
  "DE22110101111111111111111",
  "20.01.2020",
  "-35,00 EUR",
  "iShsII-Core MSCI Europe U.ETF Registered Shares o.N. in Wertpapierrechnung.",
  "Lagerland: Grossbritannien",
  "Diese Abrechnung wird maschinell erstellt und daher nicht unterschrieben.",
  "Sofern keine Umsatzsteuer ausgewiesen ist, handelt es sich gem. § 4 Nr. 8 UStG um eine umsatzsteuerfreie",
  "Leistung."
]
```

**Comment by codeofsumit** [comment]
cc @zirkler

**Comment by zirkler** [comment]
I'm gonna take care tonight 👍

**Comment by zirkler** [comment]
So this was a Sparplan buying, which I never did so I did not have any test material here.
Sparplan Buyings are now supported: see #18

**Comment by codeofsumit** [comment]
Fixed by #19
